src/emulator/helpRVL.c

```diff
@@ -949,53 +933,13 @@ s32 fn_8005F7E4(HelpMenu* pHelpMenu) {
             }
 
             if (lbl_8025D0C8 != NULL) {
-                Rect rect = {0};
-                f32 x1;
-                f32 y1;
-                f32 y0;
-                f32 x0;
-
-                rect.x1 = sRenderMode->fbWidth / 2;
-                rect.y1 = sRenderMode->xfbHeight / 2;
-
-                GXInvalidateVtxCache();
-                GXInvalidateTexAll();
-
-                if (fn_8007FC84()) {
-                    fWidth = ((sRenderMode->viWidth - 704) * 320) / 1408.0f;
-                    fHeight = ((sRenderMode->viHeight - 574) * 287) / 1148.0f;
-                    C_MTXOrtho(matrix44_3, -fHeight, fHeight + 287.0f, -fWidth, fWidth + 320.0f, 0.0f, -1.0f);
-                } else {
-                    fWidth = ((sRenderMode->viWidth - 704) * 320) / 1408.0f;
-                    fHeight = ((sRenderMode->viHeight - 480) * 240) / 960.0f;
-                    C_MTXOrtho(matrix44_3, -fHeight, fHeight + 240.0f, -fWidth, fWidth + 320.0f, 0.0f, -1.0f);
-                }
-
-                GXSetViewport(0.0f, 0.0f, sRenderMode->fbWidth, sRenderMode->efbHeight, 0.0f, 1.0f);
-                GXSetProjection(matrix44_3, GX_ORTHOGRAPHIC);
-
-                helpMenuSetupRender();
-
-                GXBegin(GX_QUADS, GX_VTXFMT0, 4);
-                x0 = rect.x0;
-                y0 = rect.y0;
-                x1 = rect.x1;
-                y1 = rect.y1;
-                GXPosition3f32(x0, y0, 0.0f);
-                GXTexCoord2u8(0, 0);
-                GXPosition3f32(x0, y1, 0.0f);
-                GXTexCoord2u8(0, 1);
-                GXPosition3f32(x1, y1, 0.0f);
-                GXTexCoord2u8(1, 1);
-                GXPosition3f32(x1, y0, 0.0f);
-                GXTexCoord2u8(1, 0);
-                GXEnd();
+                helpMenuSetupRender(&sTexObj);
             }
 
             if (fn_8007FC84()) {
-                C_MTXOrtho(matrix44_4, 3.71875f, -3.72625f, -3.8125f, 3.8125f, 0.0f, 3.9882812f);
+                C_MTXOrtho(matrix44_4, 240.0f, -243.84001f, -320.0f, 320.0f, 0.0f, 500.0f);
```

Comment:
does it make sense to use the macros here? for 320 and 240 (same for the else block)